crates/bevy_app/src/sub_app.rs

```diff
@@ -174,6 +152,28 @@ impl SubApp {
         self
     }
 
+    /// See [`App::contains_resource`].
+    pub fn contains_resource<R: Resource>(&self) -> bool {
```

Comment:
These methods shouldn't exist IMO, and definitely shouldn't be in this PR.

Reply:
Yep, I'd prefer that in a follow-up (or I suppose precursor) PR. It deserves its own consideration and shouldn't block this work.

Reply:
[Here](https://github.com/bevyengine/bevy/pull/13425/files)